### PR TITLE
[VTP-1702] Support [tag] in env pattern

### PR DESCRIPTION
**Pull request by kayakyakr**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "contentful-migrations-action",
-  "version": "2.0.0",
+  "version": "2.1.0",
```
Comment (steenzout) [inline package.json:3]: this version already exists https://github.com/the-infatuation/contentful-action/releases/tag/v2.1.0

**Review by steenzout — COMMENTED** [review]
LGTM.

@kayakyakr check the `version` number.

File: package.json (hunk 1 above)
Comment (steenzout) [reply package.json:3]:
```suggestion
  "version": "2.1.2",
```

**Review by steenzout — APPROVED** [review]

File: package.json (hunk 1 above)
Comment (kayakyakr) [reply package.json:3]: 😱

**Pull request merged**